spec.md

```diff
@@ -74,13 +109,44 @@ can be found under the "omero" key in the group-level metadata:
 }
 ```
 
-
 See https://docs.openmicroscopy.org/omero/5.6.1/developers/Web/WebGateway.html#imgdata
 for more information.
 
+### "masks"
+
+The special group "masks" found under an image Zarr contains the key `masks` containing
+the paths to mask objects which can be found underneath the group:
+
+```
+{
+  "masks": [
+    "orphaned/0"
+  ]
+}
+```
+
+Unlisted groups MAY be masks.
+
+### "color"
+
+The `color` key defines an image that is "labeled", i.e. every unique value in the image
```

Comment:
We also export color with ```style=--split```. The mask only has `1` values (not labelled) but it does have a color.

Reply:
Hmmm.... reading it, I'm not sure just having only one label makes something not labeled. The example also only shows one value ;)

Reply:
Hmm, so both ```--style=labelled``` and ```--style=split``` produce masks that are labelled. Maybe this is a bit less confusing if we ignore the ```--style=labelled``` option (since that is the default). If we also consider removing the non-compliant `6d` option then all masks are "labelled". Are any masks NOT "labelled"? So I think you can remove that whole sentence "The `color` key defines an image that is labeled, i.e. every unique value in the image represents a unique, non-overlapping object within the image." since even without any 'color', that statement is still true.